ImperatorToCk2/src/main/java/ImperatorToCK2/Output.java

```diff
@@ -1,752 +1,805 @@
+    }
+
+    public static String btitleCreation(String name, String Directory, int id) throws IOException
+    {
+
+        String VM = "\\"; 
+        VM = VM.substring(0);
+        char VMq = '"';
+        String tab = "	";
+        String oldName = name;
+        name = name.toLowerCase();
+        name = name.replace(' ','_');
+
+        if (id == 103) { //Leon in Brittany and Spain have the same name in definition.csv
+            name = "french_leon";  
+        }
+
+        Directory = Directory + VM + "common" + VM + "landed_titles";
+
+        String ck2CultureInfo ="a";   // Owner Culture Religeon PopTotal Buildings
+
+        FileOutputStream fileOut= new FileOutputStream(Directory + VM + "b_" + name + ".txt");
+        PrintWriter out = new PrintWriter(fileOut);
+
+        int flag = 0;
+
+        out.println ("c_"+name+"={");
+        out.println (tab+"b_"+name+"={");
+        out.println (tab+"}");
+        out.println ("}");
+        out.flush();
+        fileOut.close();
+
+        return ck2CultureInfo;
+    }
+
+    public static String localizationCreation(String[] name, String title, String rank, String Directory) throws IOException
+    {
+
+        String VM = "\\"; 
+        VM = VM.substring(0);
+        char VMq = '"';
+
+        ArrayList<String> oldFile = new ArrayList<String>();
+
+        oldFile = Importer.importModLocalisation(Directory);
+
+        Directory = Directory + VM + "localisation";
+        String ck2CultureInfo ="a";   // Owner Culture Religeon PopTotal Buildings
+
+        FileOutputStream fileOut= new FileOutputStream(Directory + VM + "converted_title_localisation.csv");
+        PrintWriter out = new PrintWriter(fileOut);
+
+        int flag = 0;
+        int aqq = 0;
+
+        try {
+
+            while (flag == 0) {
+                out.println (oldFile.get(aqq));
+                aqq = aqq + 1;
+
+            }
+
+        }catch (java.lang.IndexOutOfBoundsException exception){
+            flag = 1;
+
+        } 
+
+        out.println (rank+"_"+title+";"+name[0]+";"+name[0]+";"+name[0]+";;"+name[0]+";;;;;;;;;x");
+        out.println (rank+"_"+title+"_adj"+";"+name[1]+";"+name[1]+";"+name[1]+";;"+name[1]+";;;;;;;;;x");
+        out.flush();
+        fileOut.close();
+
+        return ck2CultureInfo;
+    }
+
+    public static String localizationBlankFile(String Directory) throws IOException
+    {
+
+        String VM = "\\"; 
+        VM = VM.substring(0);
+        char VMq = '"';
+
+        Directory = Directory + VM + "localisation";
+        String ck2CultureInfo ="a";   // blank default
+        FileOutputStream fileOut= new FileOutputStream(Directory + VM + "converted_title_localisation.csv");
+        PrintWriter out = new PrintWriter(fileOut);
+
+        out.println ("#Localization for all kingdom titles");
+        out.flush();
+        fileOut.close();
+
+        return ck2CultureInfo;
+    }
+    
+    public static void logPrint(String name) throws IOException //outputs to log.txt
```

Comment:
Unfortunately this will clash with the logger stuff I added. :-(

I'd suggest going with what I added for two reasons:
1 - It uses a standard java logging library, and standard is preferred.
2 - It matches the format the fronter expects, whereas this would need some more work to do so.

It's good work though. Sorry for doing something at the same time without checking!

Reply:
You can replace calls to logPrint with one of:
```
LOGGER.severe() for things that'll break the converter run entirely (or just throw an uncaught exception for it to be logged automatically)
LOGGER.warning() for bad things that you want users to be aware of, but that don't prevent the conversion from finishing
LOGGER.info() for useful information to have in the log (such as switch from one phase of conversion to another)
LOGGER.config() for debug messages. Not useful now, but handy in the future if you add a debug mode option.
LOGGER.finest() to move the status bar in the fronter. Give it things like "25%" or "75%"
```

Reply:
That sounds good, I'll replace those logPrints with LOGGER sometime tomorrow